packages/react-components/react-toast/src/components/ToastTitle/useToastTitle.tsx

```diff
@@ -43,12 +45,13 @@ export const useToastTitle_unstable = (props: ToastTitleProps, ref: React.Ref<HT
       action: 'div',
     },
     media: resolveShorthand(props.media, { required: !!intent, defaultProps: { children: defaultIcon } }),
-    intent,
     root: getNativeElementProps('div', {
       ref,
       children: props.children,
       id: titleId,
       ...props,
     }),
+    intent,
+    appearance,
```

Comment:
nit: You could just use the contexts hooks on the style hook instead of replicating the data through state.